temporian/implementation/numpy/operators/test/moving_max_test.py

```diff
@@ -58,14 +59,35 @@ def test_cc_wo_sampling(self):
     def test_cc_w_sampling(self):
         assert_array_equal(
             operators_cc.moving_max(
-                _f64([0, 1, 2, 3, 5, 20]),
-                _f32([nan, 10, nan, 12, 13, 14]),
-                _f64([-1, 3, 40]),
+                _f64([0, 1, 2, 3, 5, 20]),  # timestamps
+                _f32([nan, 10, nan, 12, 13, 14]),  # feature
+                _f64([-1, 3, 40]),  # sampling
                 3.5,
             ),
             _f32([nan, 12, nan]),
         )
 
+    def test_cc_wo_sampling_w_variable_winlength(self):
+        assert_array_equal(

```

Comment:
Some ideas of things to test (here, or in one of the other functions) :)

- Duplicate timestamps with same window length.
- Duplicate timestamps with different window length.
- Duplicate timestamps where some nan window length.
- Empty arrays
- A single value with NaN window length.
- A single value with NaN value and finite window length
- A single NaN value with NaN window length.

Reply:
added the ones not covered in existing ones